zqd/zeek/zeek.go

```diff
@@ -12,16 +12,6 @@ import (
 	"strings"
 )
 
-// ExecScript will be fed into a launched zeek process as the --exec option. The
-// default script disables the packet_filter and loaded scripts logs. These logs
-// are disabled because the emit either timeless logs or logs with timestamp
-// set to execution time rather than time of capture.
-var ExecScript = `
-event zeek_init() {
-	Log::disable_stream(PacketFilter::LOG);
-	Log::disable_stream(LoadedScripts::LOG);
-}`
-
```

Comment:
Given that we're getting rid of this, is the api able to handle timeless points? Do we have a test for this?

Reply:
We're still using this setup, but it's moved (like the zeek command line arguments) into the zeekrunner script; see https://github.com/brimsec/zeek/pull/23.